variables.tf

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "regions" {
+  description = "A list of GCP region codes for each of the given regions. i.e. [\"us-central1\"] => {\"us-central1\"=\"usc1\"}"
```

Comment:
Isn't this just a list of GCP region codes to compute.

For what it's worth, I'm not sure this module really needs inputs at all? We could simply output all the regions each time.

Reply:
Is there a change you'd like me to make here? Or an issue for later I should make?

Reply:
I'd like you to make the change here (remove the `regions` input.)

Reply:
Done